### PR TITLE
feat: sku and domain in import order

### PR DESCRIPTION
**Pull request by joao-conde**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-core/issues/4798 |
| Dependencies | -- |
| Decisions | -- |
| Animated GIF | -- |

### CODE REVIEW

File: src/js/api/order.js [hunk 1]
```diff
@@ -2456,6 +2458,8 @@ ripe.Ripe.prototype._importOrder = function(ffOrderId, options = {}) {
     if (Object.keys(parts).length > 0) contents.parts = parts;
     if (size) contents.size = size;
     if (factory) contents.factory = factory;
+    if (domain) contents.domain = domain;
```
Comment (hugo-gomes) [inline src/js/api/order.js:2461]: try to sort the const declarations and ifs to be the same, meaning
if we're declaring `domain` and `sku` before `model`, lets add their if clauses before the `model`'s if clause

**Review by hugo-gomes — CHANGES_REQUESTED** [review]

File: src/js/api/order.js (hunk 1 above)
Comment (joao-conde) [reply src/js/api/order.js:2461]: Likewise, we are declaring it after factory. The order is already wrong. I was trying not to change to many lines in this small PR

Comment (hugo-gomes) [reply src/js/api/order.js:2461]: Can you fix the ordering on a separate PR please?

**Review by hugo-gomes — APPROVED** [review]

File: src/js/api/order.js (hunk 1 above)
Comment (hugo-gomes) [reply src/js/api/order.js:2461]: Thanks

**Pull request merged**